src/lib/deploy.js

```diff
@@ -137,7 +137,7 @@ async function deploy({ alias, yes }) {
     return { smartContracts };
   });
 
-  // Identify which smart contract should be deployed for this deploy alias.
+  // Identify which smart contract to be deployed for this deploy alias.
```

Comment:
we all want to use "should" but in technical situations, we want to avoid 
see https://developers.google.com/style/word-list#letter-s 
![image](https://github.com/o1-labs/zkapp-cli/assets/10067215/eac1927b-634a-4b43-a906-b72c3ad9196c)